src/Fixer/Import/OrderedImportsFixer.php

```diff
@@ -21,13 +22,104 @@
 /**
  * @author Sebastiaan Stok <s.stok@rollerscapes.net>
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ * @author Darius Matulionis <darius@matulionis.lt>
  * @author SpacePossum
  */
 final class OrderedImportsFixer extends AbstractFixer
 {
-    const IMPORT_TYPE_CLASS = 1;
-    const IMPORT_TYPE_CONST = 2;
-    const IMPORT_TYPE_FUNCTION = 3;
+    const IMPORT_TYPE_CLASS = 'class';
+
+    const IMPORT_TYPE_CONST = 'const';
+
+    const IMPORT_TYPE_FUNCTION = 'function';
+
+    const SORT_ALPHA = 'alpha';
+
+    const SORT_LENGTH = 'length';
+
+    /**
+     * Array of import types by which use statements should be ordered. If not defined in configuration only sort algorithm will be applied.
+     *
+     * @var null|int[]
+     */
+    private static $sortTypesOrder = null;
+
+    /**
+     * Array of supported sort types in configuration.
+     *
+     * @var string[]
+     */
+    private static $supportedSortTypes = array(self::IMPORT_TYPE_CLASS, self::IMPORT_TYPE_CONST, self::IMPORT_TYPE_FUNCTION);
+
+    /**
+     * Default sort algorithm.
+     *
+     * @var string
+     */
+    private static $sortAlgorithm = self::SORT_ALPHA;
+
+    /**
+     * Array of supported sort algorithms in configuration.
+     *
+     * @var string[]
+     */
+    private static $supportedSortAlgorithms = array(self::SORT_ALPHA, self::SORT_LENGTH);
+
+    /**
+     * Configuration settings for sort algorithm and sor types.
+     *
+     * array['sortAlgorithm']   string defines sort algorithm
+     * array['typesOrder']      int[]|null defines import types order or null to sort only by algorithm
+     *
+     * @param array $configuration
```

Comment:
yep. check the docs of parent class/interface.
null is meaningful here. it is supposed to reset fixer to use default settings.